Yosemite/YosemiteTests/Stores/ProductStoreTests.swift

```diff
@@ -991,9 +1000,11 @@ final class ProductStoreTests: XCTestCase {
 
         network.simulateError(requestUrlSuffix: "products/\(sampleProductID)", error: NetworkError.notFound)
         let product = sampleProduct()
-        let action = ProductAction.updateProduct(product: product) { (product, error) in
-            XCTAssertNotNil(error)
-            XCTAssertNil(product)
+        let action = ProductAction.updateProduct(product: product) { result in
```

Comment:
and here 😉

Reply:
updated in https://github.com/woocommerce/woocommerce-ios/pull/2533/commits/9f6c5b1d6666bbd75ba2fa8564cbd4bcd2bd93a7